rdrf/rdrf/management/commands/check_logins.py

```diff
@@ -0,0 +1,132 @@
+import sys
+from datetime import datetime, timedelta
+from django.core.management import BaseCommand
+from rdrf.models import Registry
+from rdrf.reminders import ReminderProcessor
+
+from registry.groups.models import CustomUser
+from registry.patients.models import Patient
+from registry.patients.models import ParentGuardian
+
+def send_reminder(user, registry_model,process_func=None):
+    if process_func:
```

Comment:
The conditional isn't needed if you just use:

`ReminderProcessor(user, registry_model, process_func)`